superbit_lensing/medsmaker/scripts/process_mocks.py

```diff
@@ -25,64 +25,75 @@
                     help='Output directory for MEDS file')
 parser.add_argument('--fname_base', action='store', type=str, default=None,
                     help='Basename of mock image files')
+parser.add_argument('-psf_mode', action='store', choices=['piff', 'psfex'],default='piff',
+                    help='model exposure PSF using either piff or psfex')
 parser.add_argument('--meds_coadd', action='store_true', default=False,
                     help='Set to keep coadd cutout in MEDS file')
 parser.add_argument('--clobber', action='store_true', default=False,
                     help='Set to overwrite files')
-parser.add_argument('--source_select', action='store_true', default=False,
+parser.add_argument('--source_select', action='store', default=False,
```

Comment:
This should be changed to `action='store_true'`

Reply:
done